src/js/worker/core/player.js

```diff
@@ -951,14 +1013,15 @@ function generate(
         valueWithContract: 0,
     };
 
-    const rand = Math.random();
-    if (rand < 0.5) {
-        p.hgt += 1;
-    } else if (rand < 0.75) {
-        p.hgt += 2;
-    } else if (rand < 0.77) {
-        p.hgt += 3;
-    }
+	// No longer needed now that heights are generated by a distribution
```

Comment:
Yeah.  That needs to happen.  Just wanted to get some feedback before I got into this too far.